### DISCUSSION TITLE
QImg does not start load image

### DISCUSSION THREAD

**Discussion by mirkofisic** [quasarframework/quasar#11299] [CLI - SSR mode]
I use image from quasar example but it seems that the q-img component never even started loading the image, I added listeners also to check what is going on but event is never trigered, also in inspect element in network tab i don't see the request for a picture being made at all. q-img compoenent is placed inside q-page. Any advice?

`<q-img @load="onLoadImg" @error="onLoadErrorImg" src="https://placeimg.com/500/300/nature"></q-img>`

**Comment by metalsadman** [comment]
it's working fine in the docs examples or in my project. check for error logs in your browser.

**Reply by mirkofisic** [reply]
There is no any error in browser logs

**Comment by rstoenescu** [comment]
Make sure that the QImg element is rendered with at least a few pixels (width AND height).

**Reply by mirkofisic** [reply]
still does not work I set width 200px and height 200px

**Reply by mirkofisic** [reply]
Ii place <q-img @load="onLoadImg" @error="onLoadErrorImg" src="https://placeimg.com/500/300/nature"></q-img>in layout it works but in page does not work.

**Reply by rstoenescu** [reply]
Supply a codepen so we can see what's going on.

**Comment by MilosPaunovic** [comment]
It would be best to create a Codepen reproduction of your issue, so someone could help you solve the problem easier.

**Reply by mirkofisic** [reply]
problem was if I call async function with await inside <script async setup> I dont know why but this is definitely problem

**Reply by metalsadman** [reply]
> problem was if I call async function with await inside <script async setup> I dont know why but this is definitely problem

check the note https://v3.vuejs.org/api/sfc-script-setup.html#top-level-await
